examples/todo-list/test/acceptance/todo-list.acceptance.ts

```diff
@@ -37,93 +37,89 @@ describe('Application', () => {
-      .patch('/todo-lists')
-      .send(patchedColorTodo)
-      .expect(200);
-    expect(response.body).to.eql(todoLists.length);
-    const updatedTodoLists = await todoListRepo.find();
-    for (const todoList of updatedTodoLists) {
-      expect(todoList.color).to.eql(patchedColorTodo.color);
-    }
-  });
+    it('counts todoLists', async () => {
+      const response = await client
+        .get('/todo-lists/count')
+        .send()
+        .expect(200);
+      expect(response.body).to.eql(persistedTodoLists.length);
+    });
 
-  it('deletes all todoLists', async () => {
-    await givenTodoListInstance();
-    await givenTodoListInstance({title: 'so many things to do wow'});
-    await client
-      .del('/todo-lists')
-      .send()
-      .expect(200);
-    expect(await todoListRepo.find()).to.be.empty();
-  });
+    it('finds all todoLists', async () => {
+      const response = await client
+        .get('/todo-lists')
+        .send()
+        .expect(200);
+      expect(response.body).to.containDeep(persistedTodoLists);
+    });
 
-  it('gets a todoList by ID', async () => {
-    const todoList = await givenTodoListInstance();
-    const result = await client
-      .get(`/todo-lists/${todoList.id}`)
-      .send()
-      .expect(200);
-    // Remove any undefined properties that cannot be represented in JSON/REST
-    const expected = JSON.parse(JSON.stringify(todoList));
-    expect(result.body).to.deepEqual(expected);
-  });
+    it('updates all todoLists', async () => {
+      const patchedColorTodo = {color: 'purple'};
+      const response = await client
+        .patch('/todo-lists')
+        .send(patchedColorTodo)
+        .expect(200);
+      expect(response.body).to.eql(persistedTodoLists.length);
+      const updatedTodoLists = await todoListRepo.find();
+      for (const todoList of updatedTodoLists) {
+        expect(todoList.color).to.eql(patchedColorTodo.color);
+      }
+    });
 
-  it('updates a todoList by ID ', async () => {
-    const todoList = await givenTodoListInstance();
-    const updatedTodoList = givenTodoList({
-      title: 'A different title to the todo list',
+    it('deletes all todoLists', async () => {
+      await client
+        .del('/todo-lists')
+        .send()
+        .expect(200);
+      expect(await todoListRepo.find()).to.be.empty();
     });
-    await client
-      .patch(`/todo-lists/${todoList.id}`)
-      .send(updatedTodoList)
-      .expect(200);
-    const result = await todoListRepo.findById(todoList.id);
-    expect(result).to.containEql(updatedTodoList);
   });
 
-  it('deletes a todoList by ID', async () => {
-    const todoList = await givenTodoListInstance();
-    await client
-      .del(`/todo-lists/${todoList.id}`)
-      .send()
-      .expect(200);
-    await expect(todoListRepo.findById(todoList.id)).to.be.rejectedWith(
-      /no TodoList found with id/,
-    );
+  context('when dealing with a single persisted todoList', () => {
```

Comment:
suggestion: perhaps the `single persisted todoList` tests can come first than the tests for `multiple persisted todoLists`?

Reply:
I think it can, but in that case I'd also like to change the ordering of the methods in the generated CLI as well. I'll leave that up to another task if that's something we want to do

Reply:
The current ordering:
1. create
2. count, (find/delete/replace)All
3. (find/delete/replace)ById

I want to switch 2 and 3 around so that it goes from operation on singular entity to multiple

Reply:
Yeah that'd be good. I'm okay with a follow up task for it.